logging/samples/listlogs.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Command-line program to list the logs in a Google Cloud Platform project.
+
+Simple command-line program to demonstrate connecting to the Google Cloud
+Logging API to retrieve logs data, using application default credentials[1] to
+authenticate.
+
+This application will run--without modification--on 1) Google App Engine,
```

Comment:
We recommend users running this locally to use ``gcloud auth``. Also, no need to have the entire ADC description here, just give a brief summary and link them for more information.